parsers/verilog.c

```diff
@@ -116,12 +118,14 @@ static kindDefinition SystemVerilogKinds [] = {
  { true, 'A', "assert",    "assertions" },
  { true, 'C', "class",     "classes" },
  { true, 'V', "covergroup","covergroups" },
+ { true, 'E', "enum",      "enumerators" },
```

Comment:
`enum` kind is already defined here, but for variables to type `enum`.
The possible values that that variable can take were introduced as constants.